src/main/scala/generate/database/PostgresGenerator.scala

```diff
@@ -9,16 +9,16 @@ object PostgresGenerator extends DatabaseGenerator {
   /** Given a column, parse it into the type required by Postgres */
   private def parseColumn(column: Column): String =
     indent(
-      column match {
-        case IntColumn(name)        => s"$name INT"
-        case FloatColumn(name)      => s"$name REAL"
-        case StringColumn(name)     => s"$name TEXT"
-        case BoolColumn(name)       => s"$name BOOLEAN"
-        case DateColumn(name)       => s"$name DATE"
-        case TimeColumn(name)       => s"$name TIME"
-        case DateTimeTzColumn(name) => s"$name TIMESTAMPTZ"
-      },
-      length = 4
+      column.colType match {
+        case Some(ColType.IntCol)        => s"${column.name} INT"
+        case Some(ColType.FloatCol)      => s"${column.name} REAL"
+        case Some(ColType.StringCol)     => s"${column.name} TEXT"
+        case Some(ColType.BoolCol)       => s"${column.name} BOOLEAN"
+        case Some(ColType.DateCol)       => s"${column.name} DATE"
+        case Some(ColType.TimeCol)       => s"${column.name} TIME"
+        case Some(ColType.DateTimeTzCol) => s"${column.name} TIMESTAMPTZ"
```

Comment:
What about making a function that turns a ColType into a string, to save the replication of `column.name`?